app/views/members/_gardens.html.haml

```diff
@@ -7,31 +7,42 @@
         - first_garden = false
         = link_to g.name, "#garden#{g.id}", 'data-toggle' => 'tab'
     - if current_member == member
-      %li= link_to 'New Garden', new_garden_path
-  .tab-content
+      %li.navbar-right 
+        = link_to new_garden_path, class: 'btn' do
+          Add New Garden
```

Comment:
[Doesn't look like it](http://stackoverflow.com/questions/21415570/pass-ruby-block-to-partial-in-haml) - in general, Haml uses indentation rather than explicit `end` statements.